lib/ngx/errlog.md

```diff
@@ -210,6 +211,30 @@ clear the table yourself before feeding it into the `errlog.get_logs` function.
 
 [Back to TOC](#table-of-contents)
 
+get_sys_filtering_level
+-----------------------
+**syntax:** *log_level = log_module.get_sys_filtering_level()*
+
+**context:** *any*
+
+Return system default filtering level as an integer. Later the return value
+could be used as Nginx log level constant. For example:
+
+```lua
+-- config.lua
+local errlog = require "ngx.errlog"
+_M.log_level = error_log.get_sys_filtering_level()
+return _M
+```
+
+```lua
+-- req.lua
+local log_level = config.log_level
+errlog.set_filter_level(log_level)
```

Comment:
Why set the sys filter level to the soft level for our capturing facility? This is a no-op since the system filtering level is mandatory anyway.

We need a simpler and more meaningful code example here. The current code example is confusing.